recipes-multimedia/rpidistro-vlc/files/0001-configure-fix-linking-on-RISC-V-ISA.patch

```diff
@@ -0,0 +1,18 @@
+From: =?utf-8?q?R=C3=A9mi_Denis-Courmont?= <remi@remlab.net>
+Date: Sat, 16 Jun 2018 21:31:45 +0300
+Subject: configure: fix linking on RISC-V ISA
```

Comment:
Upstream status missing for all these patches.

Reply:
fixed: https://github.com/agherzan/meta-raspberrypi/pull/981/commits/728905704d518db38babc2bed66811abd3673284